portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,31 +14,91 @@
 
 package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 import java.io.IOException;
+import com.google.sps.data.Comment;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
+import java.util.List;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```

Comment:
nit: do you want to update the class documentation, now that it is handling comments :)

Reply:
I mean, you can update the comment to reflect what this class is now doing, something like:

`
/** Servlet that returns stores and retrieves comments from datastore. */
`

Reply:
Updated all the current classes to have more accurate documentation